FreeSurfer/FreeSurferPipeline.sh

```diff
@@ -294,6 +300,10 @@ get_options()
 				p_extra_reconall_args+="${extra_reconall_arg} "
 				index=$(( index + 1 ))
 				;;
+			--pipeline-mode=*)
+				p_pipelinemode=${argument#*=}
```

Comment:
Change to `--processing-mode` and `p_processing_mode`

Reply:
Done.